src/main/java/org/jboss/tm/ExtendedJBossXATerminator.java

```diff
@@ -63,14 +63,22 @@
      *
```

Comment:
I think we should deprecate Transaction getTransaction(Xid xid) throws XAException; as I don't think it is used anywhere, and add throws UnsupportedOperation to the IFC